src/main/webapp/history/history.js

```diff
@@ -19,51 +18,337 @@
                 tooltip.enable();
             });
         });
+		
+		function renderTrendChart(chartDivId, model, settingsDialogId, chartClickedEventHandler) {
+			const chartPlaceHolder = document.getElementById(chartDivId);
+			const chart = echarts.init(chartPlaceHolder);
+			chartPlaceHolder.echart = chart;
+			
+			const textColor = getComputedStyle(document.body).getPropertyValue('--darkreader-text--text-color') || getComputedStyle(document.body).getPropertyValue('--text-color') || '#222';
+			const showSettings = document.getElementById(settingsDialogId);
+			let darkMode = getComputedStyle(document.body).getPropertyValue('--darkreader-bg--background')
+			darkMode = darkMode !== undefined && darkMode !== null && darkMode !== ''
+			console.log('darkMode: ' + darkMode)
+			let series = model.duration.series.concat(model.result.series)
+			series.forEach(s => s.emphasis = {
+				disabled: true
+			});
+			const options = {
+				animation: false,
+				darkMode: darkMode,
+				//backgroundColor: getComputedStyle(document.body).getPropertyValue('--darkreader-bg--background') || getComputedStyle(document.body).getPropertyValue('--bs-body-bg') || '#fff',
+				toolbox: {
```

Comment:
Not sure exactly what line but on a Mac 13" screen this toolbox is being rendered off screen for me partially:

<img width="793" alt="image" src="https://github.com/jenkinsci/junit-plugin/assets/21194782/852e96f2-15af-4fba-86fa-2deaa6cb69ad">

Whereas on an existing history chart it looks fine to me, e.g. 
https://ci.jenkins.io/job/Core/job/jenkins/job/master/6294/testReport/history/

(takes awhile to load)

Reply:
should be fixed